ansible/roles/test/files/ptftests/dhcpv6_relay_test.py

```diff
@@ -347,3 +399,5 @@ def runTest(self):
         self.verify_relayed_request_relay_forward()
         self.server_send_reply_relay_reply()
         self.verify_relayed_reply()
+        self.client_send_relay_forward()
+        self.verify_relayed_relay_forward()
```

Comment:
1. you need to skip this whole file on 201911 branch images.
2. how can you make this test not failing before the buildimage change is merged?

Reply:
1. skip release here takes care of that. Will create a PR to remove 201811 skip releases https://github.com/Azure/sonic-mgmt/blob/910266eff524f6aec955bcc47b0d322cdb69d212/tests/dhcp_relay/test_dhcpv6_relay.py#L170
2. needs to merge buildimage first. converting this PR to draft.